crystal/shivneri/src/server.cr

```diff
@@ -7,6 +7,6 @@ module Server
   Shivneri.port = 3000
   Shivneri.host = "0.0.0.0"
   Shivneri.open do
```

Comment:
is the block (`do` ... `end`) required ?

Reply:
have removed